src/backend/api/Fusion.Resources.Api/Notifications/ContractAllocationNotificationHandler.cs

```diff
@@ -36,10 +36,17 @@ public async Task Handle(ContractRoleDelegated notification, CancellationToken c
             if (delegatedRole == null)
                 return;
 
-            await notificationClient.CreateNotificationAsync(n => n
-                    .WithRecipient(delegatedRole.Person.AzureUniqueId)
-                    .WithTitle($"You were delegated {delegatedRole.Type} role in {delegatedRole.Contract.ContractNumber} - {delegatedRole.Project.Name}")
-                    .WithDescriptionMarkdown(NotificationDescription.DelegateAssigned(delegatedRole)));
+            await notificationClient.CreateNotificationForUserAsync(delegatedRole.Person.AzureUniqueId,
```

Comment:
Fusion.Integration.Notification 5.1.0 required change.
Commented code can be removed.